npm/private/npm_package_store.bzl

```diff
@@ -303,10 +342,14 @@ deps of npm_package_store must be in the same package.""" % (ctx.label.package,
 
     files_depset = depset(files)
 
+    for transitive_package_store_infos_depset in transitive_package_store_infos_depsets:
+        for npm_package_store_info in transitive_package_store_infos_depset.to_list():
```

Comment:
I thought the `to_list` could be avoided when we're just putting `npm_package_store_infos` into `depset(transitive)`?

Reply:
This is an unusual case as its a depset of NpmPackageStoreInfo providers and we need to get at the underlying list so we can get at the underlying providers to pull `.transitive_files` out of them.

On the bright side, this code path is _only_ followed for js_library links which should be little to none in most cases.

The code that populates it,

```
transitive_package_store_infos_depsets.append(ctx.attr.src[JsInfo].npm_package_store_infos)
```

is on in the `elif ctx.attr.src and JsInfo in ctx.attr.src:` path